scripts/inc.Makefile

```diff
@@ -0,0 +1,53 @@
+RM         ?= rm
+CP         ?= cp
+MV         ?= mv
+
+GO         ?= go
+GOFMT      ?= gofmt
+GOLINT     ?= golint
+GOTEST     ?= go test
+GOTOOL     ?= go tool
+
+DOCKER     ?= docker
+
+LDFLAGS     =
+
+all: build
+
+help: ## Display this help
+	@awk 'BEGIN {FS = ":.*?## "; printf "\n$(PACKAGE_DESC)\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z0-9._-]+:.*?## / {printf "  \033[36m%-20s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+
+mod-download: ## Download modules using 'go mod download'
+	$(GO) mod download
+
+build: ## Build the binary using 'go build'
+	$(GO) build -ldflags "$(LDFLAGS)" -installsuffix cgo -o $(PACKAGE)
+
+install: ## Install the binary using the 'go install'
+	$(GO) install -ldflags "$(LDFLAGS)" -installsuffix cgo
+
+test: ## Run unit tests
+	$(GOTEST) -count=1 ./...
```

Comment:
From my experience `go test` caching is really clever and actually something you want to leverage as a developer. Any reason to disable it?

Reply:
i was getting conflicting results while working on #47. Depending on how I was invoking the tests `make test` vs `make -C awscodecommit test` i was seeing different results :man_shrugging:

Reply:
maybe I was doing something wrong. will remove it since it makes sense to have cached results

Reply:
Thanks! Let's investigate deeper in case it happens again.